File: Rotasi Matriks/Rotasi-Matriks.py
kolom = int (input ("Masukkan panjang kolom matriks: "))
baris = int (input ("Masukkan panjang baris matriks: "))

array = [[0] * baris for i in range (kolom)]

print ("Masukkan elemen matriks:")
for i in range (kolom):
    input_kolom = input ().split ()
    for j in range (baris):
        array [i][j] = int (input_kolom [j])

print ("Rotasi matriks:")
for j in range (baris):
    for i in range (kolom - 1, - 1, -1):
        print (array [i][j], end = " ")
    print ()

File: Rotasi Matriks/Rotasi-Matriks.c
#include <stdio.h>

int main ()
{
    int kolom = 0, baris = 0;

    printf ("Masukkan panjang kolom matriks: ");
    scanf ("%d", &kolom);
    printf ("Masukkan panjang baris matriks: ");
    scanf ("%d", &baris);

    int array [kolom][baris];

    puts ("Masukkan elemen matriks:");
    for (int i = 0; i < kolom; i++)
        for (int j = 0; j < baris; j++)
            scanf ("%d", &array [i][j]);

    puts ("Rotasi matriks:");
    for (int j = 0; j < baris; j++)
    {
        for (int i = kolom - 1; i >= 0; i--)
            printf ("%d ", array [i][j]);
        puts ("");
    }

    return 0;
}